content/board/FlorianHeigwer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Florian Heigwer"
+tagline: "Event officer"
+date: 2018-07-13
+publishdate: 1970-01-01
```

Comment:
i don't think this is accurate 🤔

Reply:
As the date I put the first time your name appeared in the Agenda and minutes

Publishdate is just there so it will not hide member if we put a date in the future. Note really needed here but okay